WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -372,6 +372,9 @@ private void endMigration() {
 
     public static void registerMigrationListener(MigrationListener listener) {
         sMigrationListener = listener;
+        if (sIsMigrationError) {
+            sMigrationListener.onError();
```

Comment:
@aforcier the parameter can still be null, annotation is ok
@maxme I generally have this idea that in order for a method to accomplish its mission, first thing to do inside is to validate it has everything needed in order to go ahead. But your call totally, this is nitpicking at this point.